Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -52,18 +52,44 @@ public ApplicationContextManagerBlazor(AuthenticationStateProvider authenticatio
 
     private void InitializeUser()
     {
-      AuthenticationStateProvider_AuthenticationStateChanged(AuthenticationStateProvider.GetAuthenticationStateAsync());
+      Task<AuthenticationState> task = default;
```

Comment:
@swegele I think this captures the fix to the issue of setting the user if it hasn't already been set?

Reply:
I continue to be surprised that this is a problem. I do wonder if it's a peculiarity of something else, such as the host page of the Blazor island, or a left over bit of an experiment that is still in place and forgotten. However, I haven't had a chance to check @swegele 's demo code, so I can't be sure.

I'd be tempted to create a new SSB solution and try the code without this change, to see if it's really still a problem.

Like I've said before, SSB never exists in isolation, as something has to host it. I'm wondering if the tyre kicking being done doesn't take this into account, but I haven't managed to make time to check myself, sorry.

Reply:
That should do it - yes